lib/evss/pciu_address/pciu_address_line_validator.rb

```diff
@@ -2,8 +2,8 @@
 
 class PciuAddressLineValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
-    unless value =~ %r(\A[-a-zA-Z0-9 \"#%&'()+,.\/:@]{1,20}\z) || value.blank?
-      record.errors[attribute] << "must match \A[-a-zA-Z0-9 \"#%&'()+,.\/:@]{1,20}\z')"
+    unless value =~ %r{\A[-a-zA-Z0-9 \"#%&'()+,.\/:@]+\z} || value.blank?
+      record.errors[attribute] << "must match \A[-a-zA-Z0-9 \"#%&'()+,.\/:@]+\z')"
```

Comment:
Unrelated to PR (and ignorable), but might I recommend a making this a bit more human-readable?  Example,

`"Contains invalid characters.  Must match \A[-a-zA-Z0-9 \"#%&'()+,.\/:@]+\z')"`

Bonus points if you could even match the invalid character(s) and list them in the error message.